package/src/simulator_sw_0701/simu.py

```diff
@@ -223,19 +220,19 @@ def preparation(self, simulation_time, output_folder, sequences_file, Metadata_f
 
         elif self.mode == 'cont':
 
-            note = '---*---*---*---*---*---*---'
-            metadata(Metadata_file, note)  
-            note = 'Continue the simulation on' + time.strftime("%a, %d %b %Y %H:%M:%S", time.gmtime())
+            note = EOL.join(['---*---*---*---*---*---*---',
+                             f"Continue the simulation on {time.strftime("%a, %d %b %Y %H:%M:%S", 
+                                                                         time.gmtime()}"])
```

Comment:
Multi-line insertions into a formatted string are a little confusing, consider changing to:

```suggestion
            note = EOL.join(['---*---*---*---*---*---*---',
                             "Continue the simulation on "])
            note += time.strftime("%a, %d %b %Y %H:%M:%S", time.gmtime())
```